src/main/kotlin/com/ampnet/identityservice/service/impl/FaucetQueueService.kt

```diff
@@ -38,4 +39,15 @@ class FaucetQueueService(
             )
         }
     }
+
+    override fun executeBlockchainTask(task: BlockchainTask): String? =
+        if (task.payload == null) {
```

Comment:
Use `task.payload?.let { ... }` instead of if-else here

Reply:
But `let` will give you the value, here is needed the case without the value

Reply:
Oh, it's the other way around. Then disregard comments about this.